tests/assets/crs/hyperband.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: kubeflow.org/v1beta1
+kind: Experiment
+metadata:
+  name: hyperband
+spec:
+  parallelTrialCount: 2
+  maxTrialCount: 2
```

Comment:
it's due to the calculations of the algorithm, specifically [this part](https://github.com/kubeflow/katib/blob/v0.15.0/pkg/suggestion/v1beta1/hyperband/service.py#L228-L232), mathemtically I think 2 is the lowest it can be.